central/graphql/resolvers/cluster_vulnerabilities.go

```diff
@@ -316,14 +316,14 @@ func withOpenShiftTypeFiltering(q string) string {
 		search.NewQueryBuilder().AddExactMatches(search.CVEType, storage.CVE_OPENSHIFT_CVE.String()).Query())
 }
 
-func (resolver *clusterCVEResolver) withClusterVulnerabilityScope(ctx context.Context) context.Context {
+func (resolver *clusterCVEResolver) clusterVulnerabilityScopeContext() context.Context {
 	if features.PostgresDatastore.Enabled() {
-		return scoped.Context(ctx, scoped.Scope{
+		return scoped.Context(resolver.ctx, scoped.Scope{
 			ID:    resolver.data.GetId(),
 			Level: v1.SearchCategory_CLUSTER_VULNERABILITIES,
 		})
 	}
-	return scoped.Context(ctx, scoped.Scope{
+	return scoped.Context(resolver.ctx, scoped.Scope{
```

Comment:
Given that the UI changes and the BE changes are dependent on Postgres, there is no longer any requirement to support new endpoints without Postgres other than for the sake of convenience to front end development (which is not necessary since the BE changes now work with Postgres enabled)